index.html

```diff
@@ -3,7 +3,27 @@
 <head>
   <title>Redirecting</title>
   <script>
-    window.location.replace("2.2");
+    const LatestVersion = "latest";
+
+    let xhr = new XMLHttpRequest();
+   
+    xhr.open("GET", "versions.json");
+    xhr.onload = function () {
+      let versions = JSON.parse(this.responseText);
+
+      let res = versions.find(it => {
+        return it["aliases"].find(alias => { return alias == LatestVersion; });
+      });
+
+      if(!res){
+        alert("Hello! " + location + " not found");
```

Comment:
The error message will make no sense. Why location is not found?

Reply:
how about: " version latest is not found!"?